dev-python/matplotlib/matplotlib-3.7.0.ebuild

```diff
@@ -4,7 +4,7 @@
 EAPI=8
 
 DISTUTILS_USE_PEP517=setuptools
-PYTHON_COMPAT=( python3_{9..11} )
+PYTHON_COMPAT=( python3_{9..11} pypy3 )
```

Comment:
The tests are failing for me:

```
FAILED tests/test_image.py::test_large_image[png-col-16777216-2\\*\\*24 rows-upper] - Failed: DID NOT WARN. No warnings of type (<class
 'UserWarning'>,) were emitted.
FAILED tests/test_image.py::test_large_image[png-col-16777216-2\\*\\*24 rows-lower] - Failed: DID NOT WARN. No warnings of type (<class
 'UserWarning'>,) were emitted.
FAILED tests/test_pickle.py::test_simple - AssertionError
FAILED tests/test_pickle.py::test_complete[png] - AssertionError
FAILED tests/test_pickle.py::test_pickle_load_from_subprocess[png] - AssertionError
FAILED tests/test_pickle.py::test_no_pyplot - AssertionError
FAILED tests/test_tightlayout.py::test_collapsed - Failed: DID NOT WARN. No warnings of type (<class 'UserWarning'>,) were emitted.
FAILED tests/test_widgets.py::test_radio_buttons[png] - matplotlib.testing.exceptions.ImageComparisonFailure: images not close (RMS 0.6
10):
FAILED tests/test_widgets.py::test_check_radio_buttons_image[png] - matplotlib.testing.exceptions.ImageComparisonFailure: images not cl
ose (RMS 1.622):
FAILED tests/test_widgets.py::test_check_buttons[png] - matplotlib.testing.exceptions.ImageComparisonFailure: images not close (RMS 0.5
73):
FAILED tests/test_widgets.py::test_check_buttons_lines[png] - matplotlib.testing.exceptions.ImageComparisonFailure: images not close (RMS 2.085):
FAILED tests/test_text.py::test_unsupported_script - AssertionError: assert [('Glyph 2534 (\\N{BENGALI DIGIT ZERO}) missing from current font.',), ('Matplotlib currently does not supp...
```

It's passing on plain CPython 3.9. 
[dev-python:matplotlib-3.7.0:20230227-160145.log](https://github.com/gentoo/gentoo/files/10841718/dev-python.matplotlib-3.7.0.20230227-160145.log)